app/models/labware_creators/stamped_plate_compressed.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module LabwareCreators
+  # This creator takes all the wells from the parent plate and compresses them to the top-left
+  # of the child plate. It uses the standard well filter.
+  class StampedPlateCompressed < StampedPlate
+    # Returns the lists of wells from the parent labware in column order, i.e.,
+    # A1, B1, ... H1, A2, B2, ... etc. This method is invoked by well_filter.
+    # The order of wells returned by the 'filtered' method of the well_filter
+    # is influenced by the order of wells returned by this method.
+    #
+    # @return [Array<Well>] the wells of the parent labware in column order.
+    #
+    def labware_wells
+      parent.wells_in_columns
+    end
+
+    # Returns an array of the filtered parent wells.
+    #
+    # @return [Array<Well>] the filtered wells of the parent
+    #
+    def parent_wells_to_transfer
+      well_filter.filtered.map(&:first)
+    end
+
+    # Returns the well object at a specified well location on a given plate.
+    #
+    # @param plate [Plate] the plate object
+    # @param well_location [String] the well location on the plate
+    # @return [Well] the well object at the well location on the plate
+    #
+    def get_well_for_plate_location(plate, well_location)
+      plate.wells.detect { |well| well.location == well_location }
+    end
```

Comment:
Recently we have noticed that we are repeating this method. It was refactored into a Plate method. 
This method definition can be removed and `child_plate.well_at_location(destination_location)` can be called in the request_hash method instead.

Reply:
Thank you, have changed to use that.